From 6e5b99f70a1d9fb51d76bb5aee6b619505e143ef Mon Sep 17 00:00:00 2001
Message-ID: <6e5b99f70a1d9fb51d76bb5aee6b619505e143ef.1721514686.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1721514686.git.sharlatanus@gmail.com>
References: <cover.1721514686.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 2 Jul 2024 13:04:38 +0100
Subject: [PATCH 17/34] gnu: python-pytest-openfiles: Update to 0.6.0.

* gnu/packages/python-check.scm (python-pytest-openfiles): Update to 0.6.0.

Change-Id: I629c0b1ad56c313a850cbca6f58e66bb1d9920d7
---
 gnu/packages/python-check.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index cb46af3490..b6e401b443 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -840,13 +840,13 @@ (define-public python-pytest-metadata
 (define-public python-pytest-openfiles
   (package
     (name "python-pytest-openfiles")
-    (version "0.5.0")
+    (version "0.6.0")
     (source
      (origin
        (method url-fetch)
-       (uri (pypi-uri "pytest-openfiles" version))
+       (uri (pypi-uri "pytest_openfiles" version))
        (sha256
-        (base32 "0n0a7fdc9m86360y96l23fvdmd6rw04bl6h5xqgl9qxfv08jk70p"))))
+        (base32 "14x9f1l9a5ghf527i5qfcfa003mkrky1dhx2hfwq5nma9v1n0lgz"))))
     (build-system python-build-system)
     (arguments
      '(#:phases
-- 
2.41.0

